fw/http_stream.c

```diff
@@ -692,6 +692,12 @@ tfw_h2_stream_send_process(TfwH2Ctx *ctx, TfwStream *stream, unsigned char type)
 	if (!stream->xmit.b_len)
```

Comment:
You should check that there is no stream->xmit.t_len == 0 here also

Reply:
if there are trailers, then the state machine input of `HTTP2_DATA` will be replaced by `HTTP2_TRAILER_HEADERS`, that is, this branch is not executable, instead, it will execute the following branch, and the latter one is the superset of the previous one.